src/admin.rs

```diff
@@ -72,12 +72,12 @@ pub struct CertDump {
     expiration_time: String,
 }
 
-#[derive(serde::Serialize, Debug, Clone)]
+#[derive(serde::Serialize, Debug, Clone, Default)]
 pub struct CertsDump {
     identity: String,
     state: String,
     // Make it an array to keep compatibility with Envoy's config_dump.
```

Comment:
is this comment still relevant?

Reply:
Whoops, forgot to add the hold label. Will send out a separate PR.